libr/core/disasm.c

```diff
@@ -856,12 +856,12 @@ static void ds_reflines_fini(RDisasmState *ds) {
 	R_FREE (ds->prev_line_col);
 }
 
-static void ds_reflines_init(RDisasmState *ds) {
+static void ds_reflines_init(RDisasmState *ds, bool json) {
 	RAnal *anal = ds->core->anal;
 
 	lastaddr = UT64_MAX;
 
-	if (ds->show_lines_bb) {
+	if (ds->show_lines_bb || json) {
```

Comment:
why not just || ds->pj? and remove this arg

Reply:
Because I'm stupid :-)